job-engine/api/src/main/java/org/eclipse/kapua/job/engine/queue/QueuedJobExecutionListResult.java

```diff
@@ -26,7 +26,7 @@
  */
 @XmlRootElement(name = "queuedJobListResult")
 @XmlAccessorType(XmlAccessType.PROPERTY)
-@XmlType(factoryClass = QueuedJobExecutionXmlRegistry.class, factoryMethod = "newQueuedJobListResult")
+@XmlType(factoryClass = QueuedJobExecutionXmlRegistry.class, factoryMethod = "newQueuedJobExecutionListResult")
 public interface QueuedJobExecutionListResult extends KapuaListResult<QueuedJobExecution> {
```

Comment:
This fix is only tangentially related to the changes in this PR: the referenced method name was probably changes in some previous refactoring. Either this class was never marshalled/unmarshalled ever since, or it was failing at runtime.